features/keychain/module/memoized-keychain.js

```diff
@@ -59,6 +59,7 @@ class MemoizedKeychain extends Keychain {
 const memoizedKeychainDefinition = {
   id: 'keychain',
   type: 'module',
+  public: true,
```

Comment:
why does this need to be public?

Reply:
it has to be public until we split into the multi-process architecture because tons of other features depend on it (message signer, tx signer, public key provider, ...)

Reply:
nm, for a second i thought memoized keychain had a different `id`